website/docs/quickstart.mdx

```diff
@@ -6,30 +6,46 @@ import Tabs from '@theme/Tabs';
 
 import TabItem from '@theme/TabItem';
 
-This guide helps you set up Unleash in just a few minutes. The fastest way to get started is by using [Docker](#set-up-unleash-with-docker). If Docker doesn't work for you, see [other ways to try Unleash](#other-ways-to-try-unleash).
+This guide helps you set up Unleash Enterprise in your own environment using [Docker](#set-up-unleash-with-docker).
+
+Alternatively, you can sign up for a [cloud-hosted trial instance](https://www.getunleash.io/pricing).
 
-## Set up Unleash with Docker
 
-### Start Unleash locally
+## Set up Unleash with Docker
 
-To start Unleash locally, clone the [Unleash repository](https://github.com/Unleash/unleash) and start the server with Docker Compose:
+To start Unleash locally, clone the Unleash repository and start the server with [Docker Compose](https://docs.docker.com/compose/):
 
 ```shell
 git clone git@github.com:Unleash/unleash.git
 
 cd unleash
-docker compose up -d
+docker compose -f docker-compose-enterprise.yml up
```

Comment:
I wonder if we should have footnote (or something) linking directly to the quickstart in the readme for OSS?

Reply:
Yes, was wondering the same, I'll add it